src/components/Message/Message.tsx

```diff
@@ -958,52 +1054,44 @@ const MessageWithContext = <
     setOverlay('message');
   };
 
+  const actionHandlers: MessageActionHandlers = {
+    deleteMessage: handleDeleteMessage,
+    editMessage: handleEditMessage,
+    reply: handleReplyMessage,
+    resendMessage: handleResendMessage,
+    showMessageOverlay,
+    toggleBanUser: handleToggleBanUser,
+    toggleMuteUser: handleToggleMuteUser,
+    toggleReaction: handleToggleReaction,
+  };
+
   const onLongPressMessage =
```

Comment:
runInJS functions aren't providing these params

Reply:
defaults handle it:

```js
            defaultHandler: payload?.defaultHandler || showMessageOverlay,
            emitter: payload?.emitter || 'message',
            event: payload?.event,

```

Reply:
ah ok event just undefined then cool